docker-compose.yml

```diff
@@ -30,30 +28,39 @@ services:
       - SONAR_TOKEN=${SONAR_TOKEN}
     environment:
       - ELASTICSEARCH_DOMAIN_URL=http://test-elasticsearch:9200
-    networks:
-      - elastic
     depends_on:
-      - kibana
-      - test-elasticsearch
-      
+      test-elasticsearch:
+        condition: service_healthy
+
   test-elasticsearch:
-    image: test-elasticsearch
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.9.3
     container_name: test-elasticsearch
-    build:
-      context: .
-      dockerfile: data/elasticsearch/Dockerfile
+    healthcheck:
+      test: [ "CMD-SHELL", "curl -fs http://localhost:9200/_cluster/health || exit 1" ]
```

Comment:
Don't think you need `|| exit 1` as it will trigger on any other non-zero code anyway.